src/types/closure.md

```diff
@@ -316,65 +316,69 @@ If a closure captures a field of a composite types such as structs, tuples, and
 ## Overall Capture analysis algorithm
 
 * Input:
-    * Analyzing the closure C yields a set of `(Mode, Place)` pairs that are accessed
+    * Analyzing the closure C yields a mapping of `Place -> Mode` that are accessed
     * Access mode is `ref`, `ref uniq`, `ref mut`, or `by-value` (ordered least to max)
+        * For a `Place` that is used in two different acess modes within the same closure, the mode reported from closure analysis is the maximum access mode.
+        * Note: `ByValue` use of a `Copy` type is seen as a `ref` access mode.
     * Closure mode is `ref` or `move`
 * Output:
-    * Minimal `(Mode, Place)` pairs that are actually captured
+    * Minimal `(Place, Mode)` pairs that are actually captured
```

Comment:
Made the change to (Place, Mode) to reflect the way information is stored and passed around in code